app/apollo/index.js

```diff
@@ -183,6 +186,13 @@ const createApolloServer = () => {
         );
       },
     },
+    onHealthCheck: async() => {
```

Comment:
`onHealthCheck` is already defined at around line 225.  Should this logic be moved down there instead?

Reply:
Merged with what was there